core/src/main/java/com/scalar/db/common/checker/OperationChecker.java

```diff
@@ -258,8 +260,8 @@ private void checkOrderings(ScanAll scanAll, TableMetadata metadata) {
     }
   }
 
-  private void checkConjunctions(ScanAll scan, TableMetadata metadata) {
-    for (Conjunction conjunction : scan.getConjunctions()) {
+  private void checkConjunctions(Selection selection, TableMetadata metadata) {
+    for (Conjunction conjunction : selection.getConjunctions()) {
```

Comment:
Changed to `Selection` since get operations will use this check as well in an upcoming PR.